src/texts/descriptions/projects.ts

```diff
@@ -1,17 +1,17 @@
-export const rhApp: Record<string, string> = {
-  projectTitle: 'RH app',
+export const facilityBooking: Record<string, string> = {
+  projectTitle: 'Facility Booking',
   projectDescription:
-    'An application aimed at bettering the lives of Raffles Hall Residents. Currently working on the design of the interface of the application.',
+    'Facility management software to deconflict usage of various venues and facilities across Raffles hall',
```

Comment:
I checked figma too and there was no full stop but I think it might be that the designers missed this out? Guessing from how the other sections have a full stop

Reply:
added a fullstop!